pkg/meta/label.go

```diff
@@ -0,0 +1,30 @@
+package meta
+
+import (
+	"github.com/giantswarm/capi-migration/pkg/project"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	versionLabel = project.Name() + ".giantswarm.io/version"
```

Comment:
I'm not sure if we should use the GS label or watch-filter label. We can change that easily.